scripts/updateSeDrivers.sh

```diff
@@ -0,0 +1,102 @@
+#! /bin/bash
+
+SCRIPT_PATH=$(dirname $(which $0))
+SCRIPT_DIR="${SCRIPT_PATH%scripts}"
+DIR_MARKER="../"
+
+DEPTH=$1
+if [[ -z ${DEPTH} ]];
+then
+    DEPTH=$(echo "${SCRIPT_DIR}" | awk -F"${DIR_MARKER}" '{print NF-1}')
+fi
+
+if [[ ${DEPTH} > 0 ]];
+then
+    TARGET_PATH=$(for i in $(seq ${DEPTH}); do echo -n ${DIR_MARKER}; done)
+else
+    TARGET_PATH=./
+fi
+
+if [[ -z ${SE_STANDALONE_VER} ]];
+then
+  SE_STANDALONE_VER=3.141.59
+fi
+
+if [[ -z ${CHROME_DRIVER_VER} ]];
+then
+    # MacOS or Linux?
```

Comment:
Does it make sense to pull the OS decision and the Chrome version decision into functions we can call from here?

Reply:
hmm. good question. i'll take a look.

Reply:
refactored script to be cleaner and less rambly